### PR TITLE
update form toggle content

### PR DESCRIPTION
**Pull request by Dhaulagiri**
### :pushpin: Summary
update form toggle content

**Preview:** https://hds-website-git-br-toggle-hashicorp.vercel.app/components/form/toggle

### :link: External links

Jira ticket: [HDS-1284](https://hashicorp.atlassian.net/browse/HDS-1284)

### CODE REVIEW

File: website/docs/components/form/toggle/partials/accessibility/accessibility.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
-#### Known Issues
+## Conformance rating
```
Comment (Dhaulagiri) [inline website/docs/components/form/toggle/partials/accessibility/accessibility.md:1]: unsure if we have conformance ratings for other variations of this component or not

**Review by jorytindall — COMMENTED** [review]
I left a bunch of small (mostly) grammatical comments, but should be good to go after addressing

**Review by KristinLBradley — APPROVED** [review]

File: website/docs/components/form/toggle/partials/guidelines/guidelines.md [hunk 2]
```diff
@@ -89,7 +42,10 @@ When using server side validation, display a Critical [AlertInline](/components/
 - We recommend keeping labels clear and concise.
 - [3.2.2 Labels or Instructions (A)](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html): Labels or instructions are provided when content requires user input.
```
[the remark this answers is not in the document]
Comment (Dhaulagiri) [reply website/docs/components/form/toggle/partials/guidelines/guidelines.md:43]: I had the same thought/question. This is a pattern in use across multiple form components that might be worth revisiting holistically to ensure we do the same throughout.

[the remark this answers is not in the document]
Comment (Dhaulagiri) [reply website/docs/components/form/toggle/partials/guidelines/guidelines.md:43]: cc @alex-ju

[the remark this answers is not in the document]
Comment (Dhaulagiri) [reply website/docs/components/form/toggle/partials/guidelines/guidelines.md:43]: Opened to track: https://hashicorp.atlassian.net/browse/HDS-1322

File: website/docs/components/form/toggle/partials/code/how-to-use.md [hunk 3]
```diff
@@ -1,18 +1,19 @@
...
+## How to use this component
 
-As mentioned above, there are different possible ways to use the `Form::Toggle` component: using the "base" variant (essentially just the control itself), using the "field" variant (the control plus label, helper text and error), or using the "group" variant (a list of fields with legend, helper text and error).
+!!! Info
 
-The "field" and "group" ones are the one that likely you will want to use, because they provide – for free and out of the box – a lot of accessibility-related functionalities. The "base" one is to be used if and when you need to achieve custom layouts or have special use cases not covered by the other variants.
+Depending on how you're going to process the user input upon submission (eg. server-side via form `POST` or client-side using JavaScript) you will need to provide a `name` attribute or a custom `ID` attribute to the field. Since the decision on how to process the input data is left to the consumers, in the examples provided we will omit these specific arguments, for sake of simplicity.
```
[the remark this answers is not in the document]
Comment (Dhaulagiri) [reply website/docs/components/form/toggle/partials/code/how-to-use.md]: unless we're sure we're going to do this across all our components I'm inclined to not accept these suggestions for the moment

File: website/docs/components/form/toggle/partials/accessibility/accessibility.md (hunk 1 above)
Comment (Dhaulagiri) [reply website/docs/components/form/toggle/partials/accessibility/accessibility.md:1]: @MelSumner do you know?

File: website/docs/components/form/toggle/partials/code/how-to-use.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (KristinLBradley) [reply website/docs/components/form/toggle/partials/code/how-to-use.md]: no worries. You can ignore if you like

**Comment by MelSumner** [comment]
I did a [PR before the holidays](https://github.com/hashicorp/design-system/pull/977) that changed all `'` to `’` where appropriate, not sure if this PR came before or after that.

**Pull request merged**